functions/Backup-DbaDatabase.ps1

```diff
@@ -440,7 +440,7 @@ function Backup-DbaDatabase {
                 Write-Message -Level Warning -Message "$failreason"
             }
 
-            $lastfull = $db.LastBackupDate.Year
+            $lastfull = ($db.Refresh()).LastBackupDate.Year
```

Comment:
I think this should be two lines at best, since refresh returns null and doesn't have attributes:

```powershell
$db.Refresh()
$lastfull = $db.LastBackupDate.Year
```

But let's see if the errors go away.